### ISSUE TITLE
fieldset.U.show() only displaying positive lon values for HYCOM

### ISSUE THREAD

**Issue by MahiMahi90** [OceanParcels/parcels#537]
When calling the fieldset.U.show() for a HYCOM dataset, only the positive lon values are displayed. (Hycom Lon values range from 0 to 360 deg.)

![hycom_w_oceanparcels](https://user-images.githubusercontent.com/44847330/53047440-b4851180-3492-11e9-843c-d1b0608b4251.png)

**Comment by erikvansebille** [comment]
Thanks for reporting this! It's indeed a bug. I will fix it in a PR, but in the meantime a workaround seems to be to explicitly give the `projection` type. I.e.
```
import cartopy
fieldset.U.show(projection=cartopy.crs.PlateCarree())
```
Could you confirm that this works, @MahiMahi90 ?

**Comment by MahiMahi90** [comment]
Yep that works! Thanks!!


<img width="637" alt="screenshot 2019-02-20 at 20 30 08" src="https://user-images.githubusercontent.com/44847330/53118869-61709480-354e-11e9-889e-a25e480b6399.png">

**Comment by erikvansebille** [comment]
Fixed in #539
